lib/phoenix_live_view/channel.ex

```diff
@@ -255,6 +255,19 @@ defmodule Phoenix.LiveView.Channel do
     handle_redirect(state, command, flash, nil)
   end
 
+  def handle_info({:phoenix_live_reload, _topic, _changed_file}, %{socket: socket} = state) do
+    Phoenix.CodeReloader.reload(socket.endpoint)
+
+    new_socket =
+      Enum.reduce(socket.assigns, socket, fn {key, val}, socket ->
+        Utils.force_assign(socket, key, val)
+      end)
+
+    handle_changed(state, new_socket, nil)
+
+    {:noreply, state}
```

Comment:
`handle_changed` returns {:noreply, new_state}` so we can simply call `handle_changed` and nix this line.
```suggestion
```